docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+
+## Configuration
+
+A workflow as described above consists of a few moving parts, which are described in more details below. **However, running the `d2-utils-cypress install` command should create all required files**.
+
+### Cypress
+
+#### `./cypress.env.json`
+
+The network shim expects a few environment variables to be present. These can be set using the `--env` flag when interacting with the cypress CLI, but base values should be provided as well the `./cypress.env.json` configuration file:
+
+```json
+{
+    "dhis2Username": "YOUR_USERNAME_HERE",
+    "dhis2Password": "YOUR PASSWORD HERE",
+    "dhis2BaseUrl": "http://localhost:8080",
+    "networkMode": "live",
+    "dhis2ApiVersion": "37"
+}
+```
+
+#### `./cypress.json`
+
+Interactive runs are not formally supported by the Network Shim, because `cypress open` (interactive mode) and `cypress run` have different behaviour regarding the plugin events. However, cypress is attempting to smooth things over and by setting `experimentalInteractiveRunEvents` to true it should theoretically be possible to at at least use stub mode in interactive mode, for example:
+
+```json
+{
+    "baseUrl": "APP URL",
+    "projectId": "PROJECT ID",
+    "testFiles": "TEST FILES",
+    "experimentalInteractiveRunEvents": true,
+    "video": false
+}
+```
+
+#### `./cypress/plugins/index.js`
+
+For most DHIS2 cypress setups, using the `enableAutoLogin` setup helps managing authentication conveniently. In turn, the `enableAutoLogin` setup utility requires the `chromeAllowXSiteCookies` when using Chrome. In most cases it makes sense to always use this plugin alongside the `networkShim`.
+
+The `networkShim` plugin itself doesn't require the `config` object, but it does accept a second `options` argument. For most DHIS2 platform apps the default options will be correct so there will be no need to pass a second argument.
+
+```js
+const {
+    networkShim,
+    chromeAllowXSiteCookies,
+} = require('@dhis2/cypress-plugins')
+
+/*
+ * const networkShimOptions = {
+ *     hosts: [
+ *         'https://main-api.com',
+ *         'https://secondary-api.com',
+ *     ],
+ *     staticResources: [
+ *         '/endpoint-which-always-return-the-same-data',
+ *         '/another-endpoint-which-always-return-the-same-data',
+ *     ]
+ * }
+ */
+
+module.exports = (on, config) => {
+    networkShim(on /*, networkShimOptions */)
+    chromeAllowXSiteCookies(on, config)
+}
+```
+
+#### `./cypress/support/index.js`
+
+The `enableNetworkShim` setup accepts no arguments and in order to execute a cypress run in a DHIS2 platform app. It's recommended that most apps use the `enableAutoLogin` setup utility to manage automatic authentication, which means the file should look like this:
+
+```js
+import { enableAutoLogin, enableNetworkShim } from '@dhis2/cypress-commands'
+
+enableAutoLogin()
+enableNetworkShim()
+```
+
+### Package scripts and dependencies
+
+#### `./package.json`
```

Comment:
I'd move this to the top, to an `install` section (as I mentioned in a comment above)

Reply:
I'll keep it here because I have something else in mind for that install section you mention.